drivers/display/CMakeLists.txt

```diff
@@ -8,6 +8,7 @@ zephyr_sources_ifdef(CONFIG_SDL_DISPLAY		display_sdl.c)
 zephyr_sources_ifdef(CONFIG_DUMMY_DISPLAY	display_dummy.c)
 zephyr_sources_ifdef(CONFIG_FRAMEBUF_DISPLAY	display_framebuf.c)
 zephyr_sources_ifdef(CONFIG_ST7789V		display_st7789v.c)
+zephyr_sources_ifdef(CONFIG_ST7735R		display_st7735r.c)
```

Comment:
I think drivers/display/st7735r.c is clear enough. There is no reason to prefix files in _drivers/display_ with _display__.

Reply:
@jfischer-no I would keep it `display_st7735r.c` for consistency as majority of drivers (including none-display related drivers) follow this naming "convention" even if it could be argued that is redundant information.

Reply:
But why follow something without a reason? Religion? IMHO better to prefix all the display controller with _dc__ and characters display with... _cdc__ :smile:

Reply:
@jfischer-no I do follow your rationale but based on the majority of files in both drivers/display and other sub-dirs I think for now it is correct to keep the display_prefix.